sdk/src/wallet/storage/mod.rs

```diff
@@ -63,17 +63,21 @@ impl Storage {
     }
 
     #[allow(dead_code)]
-    async fn batch_set(&mut self, records: HashMap<String, String>) -> crate::wallet::Result<()> {
+    async fn batch_set<T: Serialize + Send>(&mut self, records: HashMap<String, T>) -> crate::wallet::Result<()> {
```

Comment:
Only problem with this is that is forces all the records to be the same type.

Reply:
yeah, that's right ... I assumed that would be the case for a batch, but you're right ... that doesn't have to be the case.

Reply:
It breaks the function, because the point of it is to set many fields of different types. That said, I think we shoulf just remove it anyway.

Reply:
Yeah, I see. Changed it back. Removing it should be a separate issue I think.